app/scripts/lib/account-tracker.js

```diff
@@ -123,6 +120,10 @@ export default class AccountTracker {
    * Starts polling with global selected network
    */
   start() {
+    this._blockTracker.once('latest', (blockNumber) => {
```

Comment:
Is this necessary? It seems like this already happens on line 130 (`networkClientId` is null, which causes the second branch of the logic in `#getCorrectNetworkClientId` to get run).